README fix

Nobody96103 opened the pull request.

Grewestr committed:
Merge pull request #47 from WSU-4110/development

Getting up-to-date

Nobody96103 committed:
Added api_calls_string

Added the api_calls_string file for one of the displays

Nobody96103 committed:
Update api_calls_string.py

Added functions for collecting and examining the game schedule. (Better than the mess I created for homework 4).

Nobody96103 committed:
Update api_calls_string.py

Added comments and slight improvements.

Nobody96103 committed:
Update api_calls_string.py

Added a function to create a list of all players, and another to gets stats from a name

Nobody96103 committed:
Update api_calls_string.py

Added a live-stats function, changed the name of the other (get_seasonal...), and modified functions to add team & player IDs where appropriate

Nobody96103 committed:
Merge pull request #59 from WSU-4110/main

Getting up to date with main

Nobody96103 committed:
Fixing README.md

Fixing the README file to have the website at the top, and not have the unnecessary components

Nobody96103 committed:
Minor changes README.md

Modified the header type of the 'Our website' portion, and added a short introduction to the website

Nobody96103 committed:
Merge branch 'main' into Rory_Jolliff

MitchellBrownNE commented on README.md:
```diff
@@ -2,10 +2,15 @@
+
+## Our Website
+Our website can be accessed here:
+https://livesportsdisplay-78ab65502a20.herokuapp.com/
```
Looks better at top so people can access it easier.

MitchellBrownNE commented on README.md:
```diff
@@ -2,10 +2,15 @@
 
 ![alt text](https://github.com/WSU-4110/Live-Sports-Display/blob/main/Logo/MMM_Logo.png)
```
I will have to change this logo to the new one as it’s still listed as one with text.

MitchellBrownNE commented on api_calls_string.py:
```diff
@@ -0,0 +1,455 @@
+
+
+api_key = 'ry27AHhE2O4WbIO24c0DE4Zt1KeMvyMx5f31taX1'
+connection = http.client.HTTPSConnection("api.sportradar.us")
```
I would remove either line 11 or 12 as there is redundancy as it’s the same connection but different var names

MitchellBrownNE commented on api_calls_string.py:
```diff
@@ -0,0 +1,455 @@
+
+
+# gets and returns player stats in a single string
+def get_season_player_stats_from_id(player_id):
```
Might want to remove this function as it’s not live stats and won’t be used.

MitchellBrownNE commented on api_calls_string.py:
```diff
@@ -0,0 +1,455 @@
+        return None
+
+#makes a csv file containing player ids and name
+def get_team_players(team_id):
```
How will it be called since users won’t know team ids

MitchellBrownNE commented on api_calls_string.py:
```diff
@@ -0,0 +1,455 @@
+            # creates file to store teams
+            teams = open(teams_file,"w")
+            writer = csv.writer(teams)
+            writer.writerow(["Team ID", "Market", "Team Name"])
```
I would combine market and team name in one line as you might have issues in future if mixing up.

MitchellBrownNE commented on api_calls_string.py:
```diff
@@ -0,0 +1,455 @@
+        try:
+            for team_id in team_IDs:
+
+                time.sleep(5) #This makes it take a while, but it stops the api from complaining about too many calls
```
You could reduce it to time.sleep(2) as its needs minimum of one second per call

MitchellBrownNE commented on api_calls_string.py:
```diff
@@ -0,0 +1,455 @@
+                    
+                    
+                    
+    # Catching exceptions #
```
I would add an exception for CSV files on each function that uses csv files to reduce crashing errors.

MitchellBrownNE commented on api_calls_string.py:
```diff
@@ -0,0 +1,455 @@
+                
+                if player[0] == player_name:
+                    if isLive: # If getting live stats, get live, else get season
+                        return get_player_stats_from_id(player[1], player[2])
```
Does this function call seasonal and live stats. I would reduce it to one per function to help with consistency.

Nobody96103 committed:
Updating home.html

Removing the manual style: it shouldn't be necessary.
Also modified the icon to 'MMM_Mini_Logo' -

Nobody96103 committed:
Typo fix

Fixed a typo:  'boarder' to 'border'

Nobody96103 committed:
Update stats_page icon logo

Changed the icon for the stats page to the mini logo

Nobody96103 committed:
Attempt to fix navigation (1)

Nobody96103 committed:
Attempt to fix navigation (2)

Nobody96103 committed:
Removing unnecessary <style>

Removing unnecessary <style>, as the style sheet should work fine

Nobody96103 committed:
Removing unnecessary <style>

Stylesheet added: manual style unnecessary

Nobody96103 committed:
Merge branch 'Rory_Jolliff' into temp

Nobody96103 committed:
Merge pull request #66 from WSU-4110/temp

Collecting changes from main: creating temp branch to avoid messing with main (just to be safe)
